rocm-clang-ocl/PKGBUILD

```diff
@@ -1,15 +1,15 @@
 # Maintainer: acxz <akashpatel2008 at yahoo dot com>
 # Contributor: Lucas Magalhães <whoisroot@national.shitposting.agency>
 pkgname=rocm-clang-ocl
-pkgver=3.1.0
-pkgrel=2
+pkgver=3.3.0
+pkgrel=1
 pkgdesc="OpenCL compilation with clang compiler."
 arch=('x86_64')
 url="https://github.com/RadeonOpenCompute/clang-ocl"
 license=('unknown')
 makedepends=('cmake' 'hcc' 'rocm-cmake')
-source=("${pkgname}-${pkgver}::http://github.com/RadeonOpenCompute/clang-ocl/archive/roc-${pkgver}.tar.gz")
```

Comment:
Why was `${pkgname}-${pkgver} removed? It is good practice to uniquely name the downloaded tar.gz

Reply:
No reason, I did know this convention and did not find it in every package. I can put it back.